pkg/util/slice/strings.go

```diff
@@ -63,3 +63,13 @@ func StringsAnyEquals(stringSlice []string, str string) bool {
 	}
 	return false
 }
+
+// StringsAnySuffix returns true if any of the strings in the slice have the given suffix.
+func StringsAnySuffix(stringSlice []string, suffix string) bool {
```

Comment:
what's the reason for adding this util (e.g. where is it needed). Also add a small test for it

Reply:
That's what I meant

Reply:
It is meant for internal usage.